docs/how-to/tensorboard.txt

```diff
@@ -28,7 +28,7 @@ To launch TensorBoard to analyze a single Determined experiment, use
    disconnecting websocket
 
 The Determined master will schedule a TensorBoard instance in the cluster. The
-Determined CLI will wait until the TensorBoard instance is running. Then it
+Determined CLI will wait until the TensorBoard instance is running, and then it
```

Comment:
non-blocking: It's more accurate to say that it will wait for data to be present in the TensorBoard view. However, I'm not sure that this detail matters very much because the TensorBoard command is not marked as Running in our system until there are metrics to visualize.